app/src/main/java/app/foodfinderapp/MainActivity.kt

```diff
@@ -69,11 +75,6 @@ class MainActivity : AppCompatActivity() {
             when (item.itemId) {
                 R.id.navigation_profile -> {
                     if (isLoggedIn()) {
-                        val currentUser = FirebaseAuth.getInstance().currentUser
```

Comment:
The toast here was used to display user login status. I moved it to it's relevant UI layer (see ProfileFragment).

Reply:
Is this even needed?  Does the user need to see his/her email address in a toast?

More than likely, this was used for debugging.  If so, better to use the debugger instead of cluttering up code.